test/unittests/mongoBackend/mongoRegisterContext_update_test.cpp

```diff
@@ -1041,11 +1041,6 @@ TEST(mongoRegisterContext_update, NotifyContextAvailability3)
           .Times(0);
   setNotifier(notifierMock);
 
-  TimerMock* timerMock = new TimerMock();
```

Comment:
Same here... Maybe the (newly added) `uInit()` call is doing this, but I prefer you to confirm.